tools/tertiary-analysis/seurat/seurat_dim_plot.xml

```diff
@@ -0,0 +1,90 @@
+<tool id="seurat_dim_plot" name="Plot dimension reduction" version="2.3.1">
```

Comment:
in the version I suggest adding '+galaxy0' after the tool version. That is used as the build number in conda.